CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_quickbooks v0.18.1
+[PR #159](https://github.com/fivetran/dbt_quickbooks/pull/159) introduces the following updates:
+
+## Bug Fixes
+- Updated `int_quickbooks__expenses_union` to include 'Other Expense' and 'Cost of Goods Sold' account types as Expense accounts in the `quickbooks__expenses_sales_enhanced` model.
```

Comment:
Did the code change after this was written? It looks like the field you are filtering on was changed rather than adding these options.

Reply:
The change to Classification is because it incorporates these other expenses in the upstream model. Reworded to make it clearer!